soc/arm/st_stm32/stm32wba/power.c

```diff
@@ -60,35 +77,80 @@ void set_mode_stop(uint8_t substate_id)
 	}
 }
 
+#if defined(CONFIG_PM_S2RAM)
+static int suspend_to_ram(void)
+{
+	LL_LPM_EnableDeepSleep();
+
+	while (LL_PWR_IsActiveFlag_ACTVOS() == 0) {
+	}
+
+	/* Select mode entry : WFE or WFI and enter the CPU selected mode */
+	k_cpu_idle();
+
+	return 0;
+}
+
+void set_mode_suspend_to_ram(uint8_t substate_id)
+{
+	ARG_UNUSED(substate_id);
```

Comment:
This function is defined and called only here, can't you just remove the parameter if it is not needed ?

Reply:
Fixed